src/plugins/intel_gna/gna_model_serial.cpp

```diff
@@ -535,6 +535,16 @@ void GNAModelSerial::Export(const GnaAllocations& allocations, std::ostream& os)
     }
 }
 
+// To support importing of <= 2.7 format where tensor_names were not present
+static constexpr uint16_t versionWithTensorNamesMajor = 2;
+static constexpr uint32_t versionWithTensorNamesMinor = 8;
+static void AppendTensorNameIfNeeded(const uint16_t versionMajor, const uint32_t versionMinor, GnaDesc& endpoint) {
+    if (versionMajor == versionWithTensorNamesMajor && versionMinor < versionWithTensorNamesMinor &&
```

Comment:
the condition in comment says version <=2.7 but in code there is  version.major ==2  and version.minor < 8
they should be consistent.